build.gradle

```diff
@@ -104,11 +104,19 @@ task exportSampleSphereBlast {
     }
 }
 
+
 task buildTestApp {
     description 'Creates the instant app test project, then builds a test app'
     def buildFunction = 'GooglePlayInstant.Samples.TestApp.Editor.TestAppBuilder.Build'
     def buildPath = "${playInstantBuildPath}/testapp"
     def logFile = file("testapp.log").getPath()
+    
+    //Specify output file path relative to the root path using -PoutputFile
+    def outputFileArg = project.hasProperty('outputFile') ? outputFile : "${playInstantBuildPath}/testApp.apk"
```

Comment:
Super nit: slightly prefer testapp.apk here and elsewhere for file name

Reply:
Yeah I was going back and forth on that one. testapp.apk it is!